mobile/examples/phi-3/android/app/src/main/java/ai/onnxruntime/genai/demo/BottomSheet.java

```diff
@@ -0,0 +1,47 @@
+package ai.onnxruntime.genai.demo;
```

Comment:
nice addition of the adjustable settings!

if there's a UI update, should we also update the screenshots in the readme?
https://github.com/microsoft/onnxruntime-inference-examples/blob/main/mobile/examples/phi-3/android/README.md